README.md

```diff
@@ -662,6 +662,14 @@ Avoid ambiguity.
 
 **Versions X** and NOT **VX** or **version X**.
 
+## Terminology
+
+### Bus
+
+We agreed that we the word `Bus` should be avoided. Some replacements incldue:
```

Comment:
I fixed the typo and added a note that the `Bus` is allowed in pages specific to older versions. Bus most of the pages should be version-agnostic and should use the new terminology.